src/main/scala/com/mwz/sonar/scala/util/util.scala

```diff
@@ -77,3 +95,23 @@ object PathUtils {
     currentWorkdirAbsolutePath.relativize(moduleAbsolutePath)
   }
 }
+
+object MetricUtils {
+
+  /**
+   * Save a new measure for the given metric.
+   */
+  def save[T <: java.io.Serializable](
```

Comment:
You may use this helper in the [**ScoverageSensor**](https://github.com/mwz/sonar-scala/blob/master/src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala#L134) too.

Reply:
I'll roll this out across the other modules, but I'd prefer to do this in a separate pr.